documentation/milestones/individual_milestones/static_site.md

```diff
@@ -98,11 +101,16 @@ Process for drafting and updating the content of the site should balance:
 
 1. Number of unique site visitors
 2. Total number of site visits
-3. Stretch: Number of form responses
 4. Uptime service
 5. [Lighthouse score](lighthouse) for the site
 6. Deployment build time
 7. Deployment/hosting costs
+8. Number of PDF downloads per NOFO prototype
```

Comment:
nit: downloads or views?

Reply:
Good call, I had assumed the click would trigger a download, but I made this "view" in case we want to host the content